Modules/graceful_errors.py

```diff
@@ -0,0 +1,64 @@
+"""
+graceful_errors.py - provides facilities for graceful error handling
+
+Users shouldn't see the actual exception raised from a command invocation,
+    rather they should see something more lighthearted and mundane
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+"""
+import random
+from typing import Dict
+from uuid import UUID
+
+from Modules.rat_board import IndexNotFreeError, RescueBoardException, RescueNotChangedException, \
+    RescueNotFoundException
+from Modules.rat_command import InvalidCommandException, NameCollisionException
+
+BY_ERROR: Dict[type(Exception), str] = {
```

Comment:
I use and have seen RuntimeError too, so  you might want to find something for that too

Reply:
Sure thing